app/ui/src/main/java/com/fsck/k9/ui/settings/account/AccountSettingsFragment.kt

```diff
@@ -332,7 +332,7 @@ class AccountSettingsFragment : PreferenceFragmentCompat(), ConfirmationDialogFr
                 getString(R.string.cancel_action)
         )
         dialogFragment.setTargetFragment(this, REQUEST_DELETE_ACCOUNT)
-        dialogFragment.show(fragmentManager, TAG_DELETE_ACCOUNT_CONFIRMATION)
+        dialogFragment.show(fragmentManager!!, TAG_DELETE_ACCOUNT_CONFIRMATION)
```

Comment:
using "!!" defeats the purpose of kotlin's null safety

Reply:
Yes, this method or some part of the method code are no longer null safe and it wouldn't compile anymore.